zboxcore/sdk/commitworker.go

```diff
@@ -216,6 +217,7 @@ func (req *CommitRequest) commitBlobber(rootRef *fileref.Ref, latestWM *marker.W
 	wm.BlobberID = req.blobber.ID
 	wm.Timestamp = timestamp
 	wm.ClientID = client.GetClientID()
+	wm.Operation = req.operation
```

Comment:
Also include this field while signing writemarker.

Reply:
Is this correct? https://github.com/0chain/gosdk/pull/571/commits/9d10650f1b23d924c589264f4d0760eb6dea3ee4